src/routes/Footer.svelte

```diff
@@ -0,0 +1,81 @@
+
+
+<body class="Mobile_Body">
+<footer class="Rectangle">
+	<button class="Next_Button">
+		<div class="Next_Button_Text">Next Step</div>
+	</button>
+</footer>
+
+</body>
+
+
+<style>
+		.Mobile_Body{
+        /* Mobile - Step 1 */
+        position: relative;
+        width: 375px;
+        height: 667px;
+        /* BG */
+        background: rgb(239, 245, 255);
```

Comment:
fyi, there are css variables defined for font family and for all colors in src/main.css

to use try
`font-family: var(--font-primary);`